requests/MetisUsdtPrice.js

```diff
@@ -0,0 +1,74 @@
+import * as Witnet from "witnet-requests"
+
+// Retrieves USDT price of METIS from HOO API
+const hoo = new Witnet.Source("https://api.hoolgd.com/open/innovate/v1/tickers/market")
+  .parseJSONMap() // Parse a `Map` from the retrieved `String`
+  .getArray("data") // Access to the `Map` object at `data` key
+  .filter(
+    new Witnet.Script([ Witnet.TYPES.MAP ])
+      .getString("symbol")
+      .match({ "METIS-USDT": true }, false)
+  )
```

Comment:
Commenting here just to stress how fricking cool this is xD

Reply:
Done in https://github.com/witnet/witnet-price-feed-examples/commit/25025c5e20cb77c6b47dc54c83330987b85f602d.